### DISCUSSION TITLE
Removing ``botorch_wrapper``

### DISCUSSION THREAD

**Discussion by rjavadi** [emdgroup/baybe#100] [Q&A]
I wanted to ask your opinion that about my idea for removing ``botorch_wrapper`` and ``botorch.test_functions``.

``botorch_wrapper`` is only used in examples, so a simple solution would be to remove this line in ``baybe.utils.__init__``:

`from baybe.utils.botorch_wrapper import *`

And import ``botorch_function_wrapper`` like this in examples:
``from baybe.utils.botorch_wrapper import botorch_function_wrapper``

Consequently, there will be no dependency on ``botorch.test_functions`` as well.

I tried this solution and it worked. Please let me know about anything I may be missing.

**Comment by Scienfitz** [comment]
sounds good to me, seems exactly the issue

**Reply by Scienfitz** [reply]
although when tryin locally via uncommenting the import in `__init__` it didnt seem to cause speedup. I also tried tune but cant get that to run on my machine, can you post the updated report from that?

**Reply by AdrianSosic** [reply]
Sounds reasonable 👍🏼 guess this should suffice

**Reply by rjavadi** [reply]
@Scienfitz 
This is the import breakdown:

![botorch-wrapper](https://github.com/emdgroup/baybe/assets/5112312/5ce16103-d46e-4864-8eec-310c863c5ba7)

**Comment by AdrianSosic** [comment]
Merged
